web/api/badges/web_buffer_svg.c

```diff
@@ -1137,6 +1146,14 @@ int web_client_api_request_v1_badge(RRDHOST *host, struct web_client *w, char *u
     }
 
     cleanup:
+    if(end){
```

Comment:
space before '{'

Reply:
I am reviewing all the files changed to put the space, I already got the rule!